src/components/grid-aware/BlockQuoteBlock/BlockQuoteBlock.js

```diff
@@ -0,0 +1,81 @@
+import Carousel from "nuka-carousel";
+import PropTypes from "prop-types";
+import React from "react";
+import s from "./BlockQuoteBlock.module.css";
+
+const BlockQuoteBlock = ({ quote, attribution }) => {
+  const quotes = [
+    { quote: "ShelterTech, Welcomes You!", attribution: "Richard Xia" },
+    { quote: "Please Donate!", attribution: "David Agustin" },
+    { quote: "Happy Holidays", attribution: "Candy Wang" },
+  ];
+
+  return (
+    <Carousel
+      renderCenterLeftControls={function () {}}
+      renderCenterRightControls={function () {}}
+    >
+      <div className={s.bleedWrapper}>
+        <div className={s.bleedMainContent}>
+          <div className={s.gridAreaMiddle}>
+            <figure>
+              <blockquote className={s.quote}>&quot;{quote}&quot;</blockquote>
+              <figcaption className={s.attribution}>
+                &mdash;{attribution}
+              </figcaption>
+            </figure>
+          </div>
+        </div>
+      </div>
+      <div className={s.bleedWrapper}>
+        <div className={s.bleedMainContent}>
+          <div className={s.gridAreaMiddle}>
+            <figure>
+              <blockquote className={s.quote}>
+                &quot;{quotes[0].quote}&quot;
```

Comment:
Could you use curly quotes instead of straight quotes? It's `&ldquo;` for &ldquo; and `&rdquo;` for &rdquo;.

Reply:
👍 Nice to know. Changed quotes.